docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -45,36 +44,35 @@ $ cd nodejs-circuit-breaker
 $ ./start-openshift.sh
 ----
 
-These commands use the Nodeshift `npm` module to install your dependencies, launch the S2I build process on your
-{OpenShiftLocal} and start the services.
+These commands use the xref:about-nodeshift[Nodeshift] `npm` module to install your dependencies, launch the S2I build process on your
+OpenShift and start the services.
 endif::circuit-breaker-nodejs[]
 
+
+
 . Check the status of your booster and ensure your pod is running.
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
 $ oc get pods -w
-NAME                            READY     STATUS      RESTARTS  AGE
+NAME                             READY     STATUS      RESTARTS   AGE
```

Comment:
I assume that since `:app-name:` is of variable length, the header will never properly align to the columns; am I right?

Reply:
correct.